packages/frontend/src/system-integration/notifications.ts

```diff
@@ -76,19 +110,62 @@ async function showNotification(
       chatId,
       messageId,
       accountId,
+      isWebxdcInfo,
     })
   } else {
     try {
-      const notificationInfo =
-        await BackendRemote.rpc.getMessageNotificationInfo(accountId, messageId)
-      const { chatName, summaryPrefix, summaryText } = notificationInfo
+      let chatName = ''
+      let summaryPrefix = ''
+      let summaryText = ''
+      let notificationInfo: T.MessageNotificationInfo | undefined
+      let icon: string | null = null
+      if (isWebxdcInfo) {
+        const relatedMessage = await BackendRemote.rpc.getMessage(
+          accountId,
+          messageId
+        )
+        if (
+          relatedMessage.systemMessageType === 'WebxdcInfoMessage' &&
+          relatedMessage.parentId
+        ) {
```

Comment:
https://github.com/deltachat/deltachat-desktop/issues/4366 states

> note, that the msg_id from the event may refer to a webxdc-instance or a webxdc-info-message

But this only handles the latter case, am I right?